ClientApp/viking-vault/src/components/styles.css

```diff
@@ -131,21 +133,31 @@
 }
 
 .side-menu-true {
-    padding: 0% 40%;
+    padding: 20% 25%;
+    align-items: center;
 }
 
 .side-menu-false{
-    padding: 2% 20%;
+    padding: 7% 20%;
 }
 
-.paragraph-true{
-    color: #fff;
+.span-true{
+    display:none;
 }
 
-.paragraph-false{
+.span-false{
     color: #09424d;
     font-weight: lighter;
 }
+
+.dashboard-true{
```

Comment:
Same as above regarding the CSS class naming of "base" + "false"/"true"

Reply:
'Tis was solved